clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -110,6 +110,30 @@ public CreateTopicsResult createTopics(Collection<NewTopic> newTopics) {
      * This operation is supported by brokers with version 0.10.1.0 or higher. The validateOnly option is supported
      * from version 0.10.2.0.
      *
+     * <p>The following exceptions can be anticipated when calling {@code get()} on the futures obtained from the
+     * {@link CreateTopicsResult#values() values()} method of the  returned {@code CreateTopicsResult}</p>
+     * <ul>
+     *     <li>{@link org.apache.kafka.common.errors.InvalidRequestException}
+     *     if duplicate topics were present in the request</li>
+     *     <li>{@link org.apache.kafka.common.errors.ClusterAuthorizationException}
+     *     if cluster authorization failed for create operation</li>
+     *     <li>{@link org.apache.kafka.common.errors.InvalidPartitionsException}
+     *     if number of partitions is less than 1</li>
+     *     <li>{@link org.apache.kafka.common.errors.InvalidReplicationFactorException}
+     *     if replication factor is less than 1 or large than number of available brokers</li>
+     *     <li>{@link org.apache.kafka.common.errors.TopicExistsException}
+     *     if topic name already exists</li>
+     *     <li>{@link org.apache.kafka.common.errors.InvalidTopicException}
+     *     if topic name collides with other topic name</li>
+     *     <li>{@link org.apache.kafka.common.errors.InvalidReplicaAssignmentException}
+     *     if some of the partitions have different number of replicas or duplicate replica assignment found</li>
```

Comment:
How about "if the proposed replica assignment is invalid.  For example if some of the partitions have different number of replicas or a duplicate replica assignment was found"